flexmeasures/data/services/sensors.py

```diff
@@ -171,6 +171,26 @@ def get_status(
     return status
 
 
+def _get_sensor_asset_relation(
+    asset: Asset,
+    sensor: Sensor,
+    inflexible_device_sensors: list[Sensor],
+) -> str:
+    """Get the relation of a sensor to an asset."""
+    relations = list()
+    if sensor.generic_asset_id == asset.id:
+        relations.append("ownership")
```

Comment:
I like "included device" better than "ownership". We use the word "ownership" already between asset and accounts.